src/main/java/com/iexec/worker/config/BlockchainAdapterConfigurationService.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2021 IEXEC BLOCKCHAIN TECH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.iexec.worker.config;
+
+import com.iexec.worker.feign.CustomBlockchainAdapterClient;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+
+import java.time.Duration;
+
+@Slf4j
+@Service
+public class BlockchainAdapterConfigurationService {
+    private final Duration blockTime;
+
+    public BlockchainAdapterConfigurationService(CustomBlockchainAdapterClient customBlockchainAdapterClient) {
+        this.blockTime = Duration.ofSeconds(customBlockchainAdapterClient.getBlockTime());
```

Comment:
What is blockTime is null?

Reply:
Fixed. We'll now provide a default block time.